Gruntfile.js

```diff
@@ -96,13 +89,22 @@ grunt.initConfig({
 		}
 	},
 
-	jshint: {
+	eslint: {
 		options: {
-			jshintrc: true
+
+			// See https://github.com/sindresorhus/grunt-eslint/issues/119
```

Comment:
Sindre was getting too many issues reported and he disabled the bug tracker... He's don it before and restored later so this issue is most likely still there. Do you want me to remove this comment? Maybe there's a version in Web Archive?